course_discovery/apps/api/filters.py

```diff
@@ -181,6 +182,28 @@ class Meta:
         fields = ('keys', 'hidden', 'license',)
 
 
+class CourseRunOrderingFilter(OrderingFilter):
+    ordering_param = 'course_runs_sort_by'
+    allowed_custom_filters = ['key', 'start']
+
+    def get_ordering(self, request, queryset, view):
+        params = request.query_params.get(self.ordering_param)
+        if params:
```

Comment:
```suggestion
        if not params:
```
This avoids nested code. @elvinquero